lighthouse-core/audits/manifest-background-color.js

```diff
@@ -29,6 +29,10 @@ class ManifestBackgroundColor extends Audit {
       category: 'Manifest',
       name: 'manifest-background-color',
       description: 'Manifest contains `background_color`',
+      helpText: 'When your PWA loads from a user\'s homescreen, the browser ' +
```

Comment:
"When your app is launched from the user's homescreen..."

FWIW, we don't use the acronym PWA anywhere in the current report.